addons/web/static/src/webclient/navbar/navbar.js

```diff
@@ -113,17 +113,18 @@ export class NavBar extends Component {
         const sections = [
             ...sectionsMenu.querySelectorAll(":scope > *:not(.o_menu_sections_more)"),
         ];
-        sections.forEach((s) => s.classList.remove("d-none"));
+        for (const section of sections) {
+            section.classList.remove("d-none");
+        }
         this.currentAppSectionsExtra = [];
-        moreDropdown.classList.add("d-none");
 
         // ------- Check overflowing sections -------
         const sectionsAvailableWidth = sectionsMenu.offsetWidth;
         const sectionsTotalWidth = sections.reduce((sum, s) => sum + s.offsetWidth, 0);
         if (sectionsAvailableWidth < sectionsTotalWidth) {
-            // Sections are overflowing, show "more" menu
-            moreDropdown.classList.remove("d-none");
-            let width = moreDropdown.offsetWidth;
+            // Sections are overflowing
+            // Initial width is harcoded to the width the more menu dropdown will take
+            let width = 46;
```

Comment:
this seems dangerous? "More" may takes more space in different languages?

Reply:
hmm, now that I think the more menu is a +, right? in that case, should be ok

Reply:
Yes, I hesitated before writing that line...
The more dropdown is always a +, so should be ok.

I changed that behaviour because I did not like the fact that before this commit, the more dropdown was always attached in the dom, even when the screen had all the necessary width (with the .d-none class).